pkg/acsengine/defaults.go

```diff
@@ -98,8 +98,10 @@ func setDefaultCerts(a *api.Properties) (bool, error) {
 
 	ips := []net.IP{firstMasterIP}
 
-	for i := 1; i < a.MasterProfile.Count; i++ {
-		ips = append(ips, net.IP{firstMasterIP[12], firstMasterIP[13], firstMasterIP[14], firstMasterIP[15] + byte(i)})
+	// Include the Internal load balancer as well
+	for i := 1; i < (a.MasterProfile.Count + 1); i++ {
+		ip := net.IP{firstMasterIP[12], firstMasterIP[13], firstMasterIP[14], firstMasterIP[15] + byte(i)}
```

Comment:
Okay, one last comment from me... here's how I'd recommend writing it:

```
masterIP4, err := firstMasterIP.To4()
if err != nil { whatever }
ip := net.IPv4(masterIP[0], masterIP4[1], masterIP4[2], masterIP4[3] + byte(i))
```

This no longer requires understanding the `net.IP` internals.